<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuroraTrackerSettingsWidget</class>
 <widget class="QWidget" name="AuroraTrackerSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>NDI Aurora Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="updateFrequencyAuroraLabel">
              <property name="text">
               <string>Update Frequency (Hz):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="updateFrequencyAuroraLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="port01AuroraLabel">
                <property name="text">
                 <string>Port 1:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="port01AuroraVirtualRomCheckBox">
                <property name="text">
                 <string>Virtual Rom?</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="port01AuroraRomFileLineEdit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="port01AuroraRomFileBrowseButton">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="port02AuroraLabel">
                <property name="text">
                 <string>Port 2:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="port02AuroraVirtualRomCheckBox">
                <property name="text">
                 <string>Virtual Rom?</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="port02AuroraRomFileLineEdit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="port02AuroraRomFileBrowseButton">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="port03AuroraLabel">
                <property name="text">
                 <string>Port 3:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="port03AuroraVirtualRomCheckBox">
                <property name="text">
                 <string>Virtual Rom?</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="port03AuroraRomFileLineEdit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="port03AuroraRomFileBrowseButton">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="port04AuroraLabel">
                <property name="text">
                 <string>Port 4:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="port04AuroraVirtualRomCheckBox">
                <property name="text">
                 <string>Virtual Rom?</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="port04AuroraRomFileLineEdit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="port04AuroraRomFileBrowseButton">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
